project/threads/templates/threads/static_templates/landing.html

```diff
@@ -15,19 +16,19 @@
     <div class="landing-bg valign-wrapper full-height">
         <div class="container landing-title">
             <div class="row">
-                <div class="landing-title-text">Discuss <span class="normal">Problems</span></div>
-                <div class="landing-title-text">Discover <span class="normal">Solutions</span></div>
-                <div class="landing-title-text">Enact <span class="landing-title-highlight normal">Change</span></div>
-                <div class="landing-title-subtext">Building a Better Democracy for the Internet Age</div>
+                <div class="landing-title-text">{% trans 'Discuss' %} <span class="normal">{% trans 'Problems' %} </span></div>
+                <div class="landing-title-text">{% trans 'Discover' %} <span class="normal">{% trans 'Solutions' %}</span></div>
+                <div class="landing-title-text">{% trans 'Enact' %} <span class="landing-title-highlight normal"> {% trans 'Change' %} </span></div>
+                <div class="landing-title-subtext">{% trans 'Building a Better Democracy for the Internet Age' %}</div>
```

Comment:
One challenge in localization is that different languages use different word orders. So, it is important to have the whole phrase available to the translator, rather than just single words. Let's just remove the inline span elements and move the texts together into a single i18n string.

```suggestion
                <div class="landing-title-text">{% trans 'Discuss Problems' %}</div>
                <div class="landing-title-text">{% trans 'Discover Solutions' %}</div>
                <div class="landing-title-text">{% trans 'Enact Change' %}</div>
                <div class="landing-title-subtext">{% trans 'Building a Better Democracy for the Internet Age' %}</div>
```